go.mod

```diff
@@ -13,7 +13,6 @@ require (
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/protobuf v1.4.3
 	github.com/googleapis/gnostic v0.3.1 // indirect
-	github.com/gorilla/mux v1.8.0
```

Comment:
Yes and because we had a couple of different route handling patterns. This specifically made it hard to follow how routes were handled and what middleware applied to each route without looking up Gorill/mux docs.

Reply:
> This repository has been archived by the owner on Dec 10, 2022. It is now read-only.

The gorilla toolkit suite of repo's have just been archived.